lua/cmake-tools/terminal.lua

```diff
@@ -611,5 +668,16 @@ end
 function _terminal.is_installed()
   return true
 end
-
+--- Handle when a terminal process exists
+---@param opts any
+---@param on_exit function|nil function to be executed on exit
+---@param close_on_exit boolean close the terminal on exit
+function _terminal.__handle_exit(opts, on_exit, close_on_exit)
```

Comment:
why not just name it to `handle_exit`?

Reply:
i copied it from toggleterm without thinking, i now see that it doesn't work with our naming scheme